schemas/concrete/MPTitles.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
It may be a better idea to have and "MPTitle" with categories. But that's for future work